conf/airframes/MC/mc_ardrone2_opticflow_swarm0.xml

```diff
@@ -0,0 +1,281 @@
+      <define name="VIDEO_CAPTURE_PATH" value="/data/ftp/internal_000/images"/>
+    </module-->
+
+    <!--module name="opticflow_hover"/-->
+  </modules>
+
+  <commands>
+    <axis name="PITCH" failsafe_value="0"/>
+    <axis name="ROLL" failsafe_value="0"/>
+    <axis name="YAW" failsafe_value="0"/>
+    <axis name="THRUST" failsafe_value="3000"/>
+  </commands>
+
+  <servos driver="Default">
+    <servo name="TOP_LEFT" no="0" min="0" neutral="1" max="500"/>
+    <servo name="TOP_RIGHT" no="1" min="0" neutral="1" max="500"/>
+    <servo name="BOTTOM_RIGHT" no="2" min="0" neutral="1" max="500"/>
+    <servo name="BOTTOM_LEFT" no="3" min="0" neutral="1" max="500"/>
+  </servos>
+
+  <section name="MIXING" prefix="MOTOR_MIXING_">
+    <define name="TRIM_ROLL" value="0"/>
+    <define name="TRIM_PITCH" value="0"/>
+    <define name="TRIM_YAW" value="0"/>
+
+    <!-- Time cross layout (X), with order NW (CW), NE (CCW), SE (CW), SW (CCW) -->
+    <define name="TYPE" value="QUAD_X"/>
+  </section>
+
+  <command_laws>
+    <call fun="motor_mixing_run(autopilot_get_motors_on(),FALSE,values)"/>
+    <set servo="TOP_LEFT" value="motor_mixing.commands[MOTOR_FRONT_LEFT]"/>
+    <set servo="TOP_RIGHT" value="motor_mixing.commands[MOTOR_FRONT_RIGHT]"/>
+    <set servo="BOTTOM_RIGHT" value="motor_mixing.commands[MOTOR_BACK_RIGHT]"/>
+    <set servo="BOTTOM_LEFT" value="motor_mixing.commands[MOTOR_BACK_LEFT]"/>
+  </command_laws>
+
+   <section name="VISION" prefix="VISION_">
+    <define name="HOVER" value="FALSE"/>
+    <define name="PHI_PGAIN" value="1500"/>
+    <define name="PHI_IGAIN" value="800"/>
+    <define name="THETA_PGAIN" value="1500"/>
+    <define name="THETA_IGAIN" value="800"/>
+    <define name="DESIRED_VX" value="0"/>
+    <define name="DESIRED_VY" value="0"/>
+   </section>
+
+  <section name="IMU" prefix="IMU_">
+    <!-- Accelero -->
+    <define name="ACCEL_X_NEUTRAL" value="2048"/>
+    <define name="ACCEL_Y_NEUTRAL" value="2048"/>
+    <define name="ACCEL_Z_NEUTRAL" value="2048"/>
+
+    <!-- Magneto calibration -->
+<!--     <define name="MAG_X_NEUTRAL" value="0"/>
+    <define name="MAG_Y_NEUTRAL" value="0"/>
+    <define name="MAG_Z_NEUTRAL" value="-180"/>
+    <define name="MAG_X_SENS" value="16." integer="16"/>
+    <define name="MAG_Y_SENS" value="16." integer="16"/>
+    <define name="MAG_Z_SENS" value="16." integer="16"/> -->
+
+<define name="MAG_X_NEUTRAL" value="3"/>
+<define name="MAG_Y_NEUTRAL" value="-15"/>
+<define name="MAG_Z_NEUTRAL" value="-39"/>
+<define name="MAG_X_SENS" value="12.1098175484" integer="16"/>
+<define name="MAG_Y_SENS" value="11.9508975917" integer="16"/>
+<define name="MAG_Z_SENS" value="12.6085474184" integer="16"/>
+
+
+    <!-- Magneto current calibration -->
+    <define name="MAG_X_CURRENT_COEF" value="0.0"/>
+    <define name="MAG_Y_CURRENT_COEF" value="0.0"/>
+    <define name="MAG_Z_CURRENT_COEF" value="0.0"/>
+
+    <define name="BODY_TO_IMU_PHI" value="0." unit="deg"/>
```

Comment:
And these... Try to only include things that are needed. Makes it easier to understand and to maintain.